deploy/podman/configmap_ssl_certs.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: certs-configmap
```

Comment:
nit: there's not much value in repeating the type in the name.

Reply:
Done.